public/app/features/dashboard/components/PanelEditor/AngularPanelPluginWarning.tsx

```diff
@@ -0,0 +1,48 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, PanelPlugin } from '@grafana/data';
+import { Alert, useStyles2 } from '@grafana/ui';
+
+export interface Props {
+  plugin: PanelPlugin;
+}
+
+export function AngularPanelPluginWarning({ plugin }: Props) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.wrapper}>
+      <Alert title="Angular panel plugin" severity="warning">
+        <div className="markdown-html">
+          <p>The selected visualization plugin is using deprecated plugin APIs.</p>
+          <ul>
+            <li>
+              <a
+                href="https://grafana.com/docs/grafana/latest/developers/angular_deprecation/"
+                className="external-link"
+                target="_blank"
+                rel="noreferrer"
+              >
+                Read more on angular deprecation
```

Comment:
Think Angular (as a proper name) is written like this:
```suggestion
                Read more on Angular deprecation
```